tools/query_verification/src/main/java/com/google/bigquery/QueryVerifier.java

```diff
@@ -72,8 +74,11 @@ public void verifyDataFree() throws InterruptedException {
     /**
      * Verifies migrated query by sending query jobs to BQ and TD to check for differences in the query results.
      */
-    public void verifyDataAware() throws InterruptedException {
+    public void verifyDataAware() throws Exception {
         List<QueryJobResults> migratedResults = migratedInstance.runQueries();
+        List<QueryJobResults> originalResults = originalInstance.runQueries();
```

Comment:
will run queries keep the same order in 2 databases?

Reply:
Assuming that the 2 query files have the queries in the same order, `migratedResults` and `originalResults` should retain that order for the query results